jupyter_server/event_schemas/kernel_actions/v1.yaml

```diff
@@ -35,11 +34,8 @@ properties:
       4. restart
          A kernel has been restarted for the given kernel id.
   kernel_id:
-    type: string
     description: |
       Kernel id.
-
-      This is a required field.
```

Comment:
Should we state the conditionality of this field?

E.g., This is required exception on failed start requests.

Reply:
Done